xml/src/main/java/org/ehcache/config/xml/ConfigurationParser.java

```diff
@@ -782,7 +782,7 @@ public int retryAttemptsDelay() {
 
     @Override
     public int minWriteDelay() {
-      return this.writebehind.getWritedelay() != null ? this.writebehind.getWritedelay().getMin().intValue() : 1 ;
```

Comment:
Isn't this encoding defaults in two places?  Something we really should try to avoid.